src/quest_manager/templates/quest_manager/category_list.html

```diff
@@ -21,7 +21,7 @@
   {% for object in object_list %}
   <tr>
     <td>{{ object.title }}</td>
-    <td>{% if object.icon %}<img src="{{ object.icon.url }}" width="50" height="50">{% endif %}</td>
+    <td>{% if object.icon %}<img class="img-responsive panel-title-img img-rounded" src="{{ object.icon.url }}" width="50" height="50">{% endif %}</td>
```

Comment:
What does "panel-title-img" do here?  I think that css sets the image size, but we've already hard coded the image size on this line.... hmm. 

I think we can use the panel-title-img class and remove the width and height attributes?  We should be using css anyway and not hardcoding.